packages/shared/lib/sdk/sync.ts

```diff
@@ -417,9 +412,24 @@ export class NangoAction {
         return (metadata['fieldMapping'] as Metadata) || {};
     }
 
-    public async log(content: string, userDefinedLevel?: UserLogParameters): Promise<void> {
+    /**
+     * Log
+     * @desc Log a message to the activity log which shows up in the Nango Dashboard
+     * note that the last argument can be an object with a level property to specify the log level
+     * example: await nango.log('This is a log message', { level: 'error' })
+     */
+    public async log(...args: any[]): Promise<void> {
+        const lastArg = args[args.length - 1];
+        const userDefinedLevel = typeof lastArg === 'object' && lastArg.level ? lastArg : undefined;
+
+        if (userDefinedLevel) {
+            args.pop();
+        }
+
+        const content = args.map((arg) => (typeof arg === 'object' ? safeStringify(arg) : String(arg))).join(' ');
```

Comment:
I would put the type check within safeStringify and handle all types, not just object but string, number, etc... in safeStringify so one can use it without thinking too much

Reply:
Unsure what you mean by adding all types but made an update here: https://github.com/NangoHQ/nango/pull/1601/commits/ceb892eaaedb1c680a2f2332cd9325fa1999b60d#diff-f8e7cae5cc6251960bb9e297501fa4e040c7266b762c60c906980b07293abe70R393